frontend/src/pages/CalendarPage/index.scss

```
@@ -2,5 +2,4 @@
   @include flex-center();
   flex-direction: column;
   justify-content: space-evenly;
```

```suggestion
  @include flex(column, space-evenly, none);
```
flex 라는 것도 mixin 으로 정의해뒀습니다.
이렇게 쓰면 더 간단할거 같아요 😃

---

앗 믹스인 함수도 있었군요! 이 부분은 이미 제가 코드를 고치고 있던 부분이 있어서 수동으로 빠르게 고쳤습니다 ㅋㅋㅋ 감사합니다!